java/src/main/native/src/ColumnViewJni.cu

```diff
@@ -15,8 +15,11 @@
  */
 
 #include <cudf/column/column_device_view.cuh>
+#include <cudf/column/column_factories.hpp>
 #include <cudf/detail/iterator.cuh>
 #include <cudf/detail/valid_if.cuh>
+#include <rmm/exec_policy.hpp>
+#include <thrust/scan.h>
```

Comment:
```suggestion

#include <rmm/exec_policy.hpp>

#include <thrust/scan.h>
```

Reply:
The clang format script deleted these two blank lines = =

Reply:
Wait, since we removed their usage, let's remove them too.

Reply:
Yeah, I am so careless T_T

Reply:
Removed them